lib/functions/general/oci-oras.sh

```diff
@@ -79,7 +79,7 @@ function run_tool_oras() {
 	# Run oras, possibly with retries...
 	if [[ "${retries:-1}" -gt 1 ]]; then
 		display_alert "Calling ORAS with retries ${retries}" "$*" "debug"
-		do_with_retries ${retries} "${ORAS_BIN}" "$@"
+		sleep_seconds="30" do_with_retries "${retries}" "${ORAS_BIN}" "$@"
```

Comment:
does this help in real life? just curious - at least can't be worse.

Reply:
It does. When ghcr.io fails, it fails for a good while, so sleeping 30 instead of 5 decreased my ghcr.io errors to almost nil.